scylla_elasticsearch_integration/query_data_from_scylla+elastic.py

```diff
@@ -4,94 +4,79 @@
 
 ### Using elasticsearch-py ###
 from cassandra.cluster import Cluster
+from cassandra import ConsistencyLevel
 from elasticsearch import Elasticsearch
 
 import random
 import argparse
-import concurrent.futures
-from cassandra import ConsistencyLevel
-from cassandra.concurrent import execute_concurrent_with_args
 
-
-## Script args and help
+## Script args and Help
 parser = argparse.ArgumentParser(add_help=True)
 
-parser.add_argument('-s', action="store", dest="SCYLLA_IP", default="127.0.0.1")
-parser.add_argument('-e', action="store", dest="ES_IP", default="127.0.0.1")
```

Comment:
This is related to any docker base configuration?

Reply:
No, I'm running scylla and elastic locally without docker containers and elastic requires you to specify the host and the port number
![image](https://github.com/user-attachments/assets/1db42315-ffb0-4bee-909e-eeb4e2e63ef2)

Reply:
Seems good! I'll test it tomorrow if no one test it before and merge. 

Cheers for the contribution!